omim2obo/main.py

```diff
@@ -163,15 +165,14 @@ def omim2obo(use_cache: bool = False):
 
         # - Non-deprecated
         omim_type, pref_label, alt_labels, inc_labels = omim_type_and_titles[omim_id]
-        label = pref_label
-        other_labels = []
-        label_endswith_included_alt, label_endswith_included_inc = False, False
+        label, other_labels, cleaned_inc_labels, label_endswith_included_alt, label_endswith_included_inc = \
```

Comment:
## Bug fix: var `cleaned_inc_labels ` referenced before assignment
Codestyle update as well: While I did that, I took the time to consolidate 5 variable declarations to a 1-liner.

Reply:
Why is this a good thing? Is this considered good practice? I personally don’t really like this, because I can’t see immediate what value each variable gets

Reply:
yeah, the readability is not good here now with this wrapping

Reply:
No problem, I'll change it now. Glad you guys are commenting on this.

I like it this way; sometimes I want more lines, and sometimes less and longer lines. Probably I should one day embrace something like `black`, though I don't like a lot of the defaults. I did just consult `black` and it also prefers 1 declaration per line (no surprise).

Reply:
Changed. I'm leaving above it as it was though. There's another tuple-unpacking multi-line declaration directly above it that I think Dazhi originally wrote:

```python
# Currently is
omim_type, pref_label, alt_labels, inc_labels = omim_type_and_titles[omim_id]

# Could change it to:
omim_type = omim_type_and_titles[omim_id][0]  # etc

# Or I could also change the `omim_type_and_titles` to a dictionary or object, e.g.
omim_type = omim_type_and_titles[omim_id]['omim_type']
```

I prefer to leave as-is though.